src/components/PlayerDash.jsx

```diff
@@ -62,6 +73,41 @@ export const PlayerDash = React.createClass({
       });
 
     });
-  }
 
+    this.firebase.getEloDateForCurrentMonth(this.props.params.playerId, 'value',(rawItems) => {
+      var items = [];
+
+      rawItems.forEach( (rawItem) => {
+        var item = rawItem.val();
+        item.id = rawItem.key();
+        items.push(item);
+      });
+
+      var getDataPoint = function getDataPoint(id, item){
```

Comment:
I think these internal functions should either be external functions on the PlayerDash object or in some other utility module. Or the raw data could be passed to your graph component and the helper functions could be properties of that. 

That said I think we can achieve the same affect using `Array.prototype.map`. This is not tested but I think should have the same effect:

```js
let data = items.map( (item) => {
        let score = item.winner == id ? item.loserNewScore : item.winnerNewScore;
        let date = new Date(item.dateTime)
        return [date, score];
      });
```

Also can we stick to using `let` or `const` again just for consistency. `const` is for constants or immutable variables. From the few articles I've read `let` is the new `var`.  Despite the fact that they have slightly different behaviour with scope and `var` is staying - I think the general concensus is `let` is an improved `var`.